src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,40 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
-    @Override
-    public void put(K key, V value) {
-    }
+    private static final int MAX_ARRAY_SIZE = 10;
+    private final Object[] keys = new Object[MAX_ARRAY_SIZE];
+    private final Object[] values = new Object[MAX_ARRAY_SIZE];
```

Comment:
It is better to have fields typed properly right away, rather than casting the return values :wink: 
```suggestion
    private final K[] keys = (K[]) new Object[MAX_ARRAY_SIZE];
    private final V[] values = (V[]) new Object[MAX_ARRAY_SIZE];
```